packages/extension-bubble-menu/src/bubble-menu-plugin.ts

```diff
@@ -74,13 +76,10 @@ export class BubbleMenuView {
     this.view.dom.addEventListener('dragstart', this.dragstartHandler)
     this.editor.on('focus', this.focusHandler)
     this.editor.on('blur', this.blurHandler)
+    this.tippyOptions = tippyOptions
+    // Detaches menu content from its current parent
+    this.element.remove()
```

Comment:
**note**: I considered safe to detach the menu element because tippy also detaches the element on mount.